File: Lua/03_conditionals.lua
-- 03_conditionals.lua
local age = 20

if age < 18 then
    print("Minor")
elseif age >= 18 and age < 65 then
    print("Adult")
else
    print("Senior")
end

local status = "active"
if status == "active" then
    print("User is active")
else
    print("Unknown")
end

File: Java/03_conditionals.java
public class Conditionals {
    public static void main(String[] args) {
        int age = 20;

        if (age < 18) {
            System.out.println("Minor");
        } else if (age >= 18 && age < 65) {
            System.out.println("Adult");
        } else {
            System.out.println("Senior");
        }

        // Switch (enhanced switch Java 14+)
        String status = "active";
        switch (status) {
            case "active" -> System.out.println("User is active");
            default -> System.out.println("Unknown");
        }
    }
}
